### DISCUSSION TITLE
conflict between Almalinux-release-8.6-0.1 and centos-linux-release-8.5-1

### DISCUSSION THREAD

**Discussion by scottin** [AlmaLinux/almalinux-deploy#152] [General]
Install failed with Almalinux 8.6 and centos 8.5
	le fichier /etc/centos-release de l'installation de almalinux-release-8.6-0.1.el8.x86_64 entre en conflit avec le fichier du paquet centos-linux-release-8.5-1.2111.el8.noarch
	le fichier /etc/redhat-release de l'installation de almalinux-release-8.6-0.1.el8.x86_64 entre en conflit avec le fichier du paquet centos-linux-release-8.5-1.2111.el8.noarch
	le fichier /etc/rpm/macros.dist de l'installation de almalinux-release-8.6-0.1.el8.x86_64 entre en conflit avec le fichier du paquet centos-linux-release-8.5-1.2111.el8.noarch
	le fichier /etc/system-release de l'installation de almalinux-release-8.6-0.1.el8.x86_64 entre en conflit avec le fichier du paquet centos-linux-release-8.5-1.2111.el8.noarch
	le fichier /etc/system-release-cpe de l'installation de almalinux-release-8.6-0.1.el8.x86_64 entre en conflit avec le fichier du paquet centos-linux-release-8.5-1.2111.el8.noarch
	le fichier /usr/lib/os-release de l'installation de almalinux-release-8.6-0.1.el8.x86_64 entre en conflit avec le fichier du paquet centos-linux-release-8.5-1.2111.el8.noarch
	le fichier /usr/share/redhat-release de l'installation de almalinux-release-8.6-0.1.el8.x86_64 entre en conflit avec le fichier du paquet centos-linux-release-8.5-1.2111.el8.noarch
	le fichier /etc/dnf/vars/contentdir de l'installation de almalinux-release-8.6-0.1.el8.x86_64 entre en conflit avec le fichier du paquet centos-linux-repos-8-3.el8.noarch

**Comment by srbala** [comment]
Translated message ...

```
Install failed with Almalinux 8.6 and centos 8.5
/etc/centos-release file of almalinux-release-8.6-0.1.el8.x86_64 installation conflicts with centos-linux-release-8.5-1.2111.el8.noarch package file
/etc/redhat-release file of almalinux-release-8.6-0.1.el8.x86_64 installation conflicts with centos-linux-release-8.5-1.2111.el8.noarch package file
/etc/rpm/macros.dist file from almalinux-release-8.6-0.1.el8.x86_64 installation conflicts with centos-linux-release-8.5-1.2111.el8.noarch package file
/etc/system-release file of almalinux-release-8.6-0.1.el8.x86_64 installation conflicts with centos-linux-release-8.5-1.2111.el8.noarch package file
/etc/system-release-cpe file from almalinux-release-8.6-0.1.el8.x86_64 installation conflicts with centos-linux-release-8.5-1.2111.el8.noarch package file
/usr/lib/os-release file of almalinux-release-8.6-0.1.el8.x86_64 installation conflicts with centos-linux-release-8.5-1.2111.el8.noarch package file
/usr/share/redhat-release file of almalinux-release-8.6-0.1.el8.x86_64 installation conflicts with centos-linux-release-8.5-1.2111.el8.noarch package file
almalinux-release-8.6-0.1.el8.x86_64 installation's /etc/dnf/vars/contentdir file conflicts with centos-linux-repos-8-3.el8.noarch package file
```
